package.json

```diff
@@ -1,12 +1,15 @@
 {
   "name": "express-task1",
   "version": "1.0.0",
-  "description": "",
-  "main": "app.js",
+  "description": "1. It will take parameters username, password, confirm password, email, id, firstname, and lastname",
+  "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
create a script for run the application and use nodemon for automatically restart the server when a file is changed